recipes/datapane/meta.yaml

```diff
@@ -0,0 +1,89 @@
+{% set name = "datapane" %}
+{% set version = "0.6.8" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: "https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz"
+  sha256: 398fdf46259e6e7485e01ddb817fe27158797120346464fbe476c80ac13f8e5b
+  
+build:
+  noarch: python
+  number: 0
+  skip: True  # [py<36]
```

Comment:
```suggestion
```

Reply:
What's the best way to specify in the recipe that the package is pure-Python but only supports python 3.6 and above ?